### PR TITLE
Adds KC Cloud Licensing Guide with GCP Marketplace

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

File: kubecost-cloud-gcp-marketplace-license.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Kubecost currently offers 30 days of Kubecost Cloud free without licensing fees.
+
+## Prerequisites
+
+* Set up a Google Cloud account with an attached [billing account](https://cloud.google.com/billing/docs/how-to/create-billing-account).
+* [Set up a Kubecost Cloud account](kubecost-cloud/cloud-installation-and-onboarding.md#creating-a-user-account)
+
+## GCP Marketplace licensing guide
+
+On the [Product details page for Kubecost](https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-cloud), select _Subscribe_. You will be taken to an Order Summary page.
```
Comment (kwombach12) [inline kubecost-cloud-gcp-marketplace-license.md:14]: @bstuder99 think some images could be useful here?

**Review by kwombach12 — CHANGES_REQUESTED** [review]

File: kubecost-cloud-gcp-marketplace-license.md (hunk 1 above)
Comment (bstuder99) [reply kubecost-cloud-gcp-marketplace-license.md:14]: I've updated the doc

**Review by kwombach12 — APPROVED** [review]
LGTM

**Review by srpomeroy — CHANGES_REQUESTED** [review]

File: kubecost-cloud-gcp-marketplace-license.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+![Kubecost Cloud product page](/images/kc-cloud-gcp.png)
+
+You should now have access to the Kubecost Cloud dashboard.
+
+## Next steps
+
+After having licensed Kubecost Cloud, you are able to install the Kubecost Cloud Agent onto all clusters you want to receive cost metrics for. See our existing [Kubecost Cloud Installation and Onboarding](/kubecost-cloud/cloud-installation-and-onboarding.md) guide for help getting started.
+
+Note that licensing of Kubecost Cloud through GCP Marketplace will not directly integrate GCP into your Kubecost Cloud environment. For more information, see our [GCP Cloud Integration guide](/kubecost-cloud/kubecost-cloud-cloud-billing-integrations/kubecost-cloud-gcp-integration.md).
```
[the remark this answers is not in the document]
Comment (srpomeroy) [reply kubecost-cloud-gcp-marketplace-license.md:34]:
```suggestion
Note that licensing of Kubecost Cloud through GCP Marketplace will not directly integrate your GCP billing data into your Kubecost Cloud environment. For more information, see our [GCP Cloud Integration guide](/kubecost-cloud/kubecost-cloud-cloud-billing-integrations/kubecost-cloud-gcp-integration.md).
```

File: kubecost-cloud-gcp-marketplace-license.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Kubecost Cloud GCP Marketplace Licensing
+
+Kubecost Cloud is [available for licensing on GCP Marketplace](https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-cloud) and can be installed in minutes. This guide will take you through licensing through GCP Marketplace, and next steps for setting up Kubecost Cloud. Kubecost currently offers 30 days of Kubecost Cloud free without licensing fees.
+
+{% hint style="info" %}
```
Comment (kwombach12) [inline kubecost-cloud-gcp-marketplace-license.md:5]: This isnt appearing right in the preview

**Review by kwombach12 — CHANGES_REQUESTED** [review]

File: kubecost-cloud-gcp-marketplace-license.md (hunk 3 above)
Comment (bstuder99) [reply kubecost-cloud-gcp-marketplace-license.md:5]: What part?

File: kubecost-cloud-gcp-marketplace-license.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* Set up a Google Cloud account with an attached [billing account](https://cloud.google.com/billing/docs/how-to/create-billing-account).
+* [Set up a Kubecost Cloud account](kubecost-cloud/cloud-installation-and-onboarding.md#creating-a-user-account)
+
+## GCP Marketplace licensing guide
+
+On the [Product details page for Kubecost](https://console.cloud.google.com/marketplace/product/kubecost-public/kubecost-cloud), select _Subscribe_. You will be taken to an Order Summary page.
+
+![Order Summary page](/images/order-summary.png)
+
+Under "1. Select Plan", the default plan should be Cloud Pro, and the default usage fee should be USD 0.167 per node per day. You can use the Pricing Calculator in the right sidebar to determine estimated costs by providing estimated timeframe of usage from 1 day to 1 year, and total node count.
```
Comment (srpomeroy) [inline kubecost-cloud-gcp-marketplace-license.md:20]: We don't typically call out pricing in docs. I suggest we remove it here.

**Review by srpomeroy — CHANGES_REQUESTED** [review]

File: kubecost-cloud-gcp-marketplace-license.md (hunk 4 above)
Comment (srpomeroy) [reply kubecost-cloud-gcp-marketplace-license.md:20]: Although it is in the images as well.

Comment (bstuder99) [reply kubecost-cloud-gcp-marketplace-license.md:20]: Yes I mention it because it is part of the Pricing Calculator included in the licensing procedure.

**Review by srpomeroy — APPROVED** [review]

**Pull request merged**